packages/react-native-gesture-handler/src/components/Pressable/StateMachine.tsx

```diff
@@ -0,0 +1,46 @@
+import { PressableEvent } from './PressableProps';
+
+interface StepDefinition {
+  signal: string;
+  callback?: (event: PressableEvent) => void;
+}
+
+class StateMachine {
```

Comment:
```suggestion
class PressableStateMachine {
```

I'd make it less general

Reply:
Done in [690fcf3](https://github.com/software-mansion/react-native-gesture-handler/pull/3571/commits/690fcf373195512689f7f3f423f43085aa4eeb09)